nservicebus/licensing/license-management.md

```diff
@@ -110,3 +110,22 @@ As a developer you can specify the license to use in your configuration code:
 
 <!-- import License -->
 
+## Where does NServiceBus look for a license?
+
+If you are experiencing an issue with licensing, it can be helpful to know where NServiceBus is looking, and in what order. An expired license in `HKEY_CURRENT_USER`, for example, will overrule a valid license in `HKEY_LOCAL_MACHINE`. Note that these vary somewhat in older versions.
```

Comment:
better to go with 

> For example an expired license in `HKEY_CURRENT_USER` will overrule a valid license in `HKEY_LOCAL_MACHINE`. 

Putting for example in the middle causes someone to rethink the purpose of the statement after they have already made assumptions about the previous part.